docs2/site/docs/guides/known-issues.md

```diff
@@ -180,6 +180,11 @@ This is done within your database queries; it is not a function of the dataloade
 `CollectionBatchDataLoader` as you would for a one-to-many relationship; then when you are loading
 data from your database within the fetch delegate, use an inner join to retrieve the proper data.
 
+## Known Issues
+
+`IResolveFieldContext.HasArgument` will return `true` for all arguments where `GetArgument` does not return `null`.
+It cannot identify which arguments have been provided a null value compared to arguments which were not provided.
```

Comment:
```suggestion
It cannot identify which arguments have been provided a `null` value compared to arguments which were not provided.
```

Reply:
Is it? The parser has this information. If you mean `field(arg1: null, arg2: 5)` vs `field(arg2: 5)`

Reply:
It is now, anyway. I can check it if was this way in 3.2

Reply:
In my opinion, this is just a flaw that can be corrected, and not some kind of fundamental design flaw.

Reply:
Agreed. That’s why it’s in “known issues” and not “faq”

Reply:
I didn't understand what you want to say. Are you documenting this behavior or will it be fixed?

Reply:
I'm documenting the behavior.  It's too much work for me to fix for 3.3.  I'll plan to fix that for 4.0.